.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# Reference: https://bit.ly/2XlscBF
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+* @sourcecred/reviewers
```

Comment:
Wondering if rather than having a team here, we should just put peoples names in directly? Right now we are implicitly assuming we will want the same set of global reviewers for all the repos, which might not be right.

Reply:
+1